contracts/test/test_rental.js

```diff
@@ -35,8 +38,8 @@ contract("Rental", accounts => {
     });
 
     it("should add active rental and get it", async () => {
-        const emit_event = await this.rentalTest.startRental(10,1,1);
-        
+       // await this.tarrifHelp.addEntry(1,10);
```

Comment:
```suggestion
       // await this.tarrifHelp.addEntry(1, 10);
```

Reply:
There should not be any lines commented like that

Reply:
Removed it, it is also called in the befre(async()) part